<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>livestock</class>
 <widget class="impgenmaint" name="livestock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>501</height>
   </rect>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="TableName">
   <string>lvstosys</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="Label4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Species and purpose</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdnew">
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/new3.png</normaloff>:/images/new3.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/delete3.png</normaloff>:/images/delete3.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hhBitBtn8">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Save the current record</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/apply.png</normaloff>:/images/apply.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hhBitBtn10">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Restore the current record</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../images.qrc">
         <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="AdvListView1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Livestock parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="Label50">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Milk production/lactation (litres):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="IMPEdit" name="Edit1">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>MILKPROD</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label51">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Lactation length (days):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="IMPEdit" name="Edit2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>LACTLENG</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label57">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Milking frequency per day (no. times per day):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="IMPEdit" name="Edit3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>MILKFREQ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Label52">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Number of lambings/kiddings per year:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="IMPEdit" name="Edit4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>NLAMB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="Label60">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Number of offpring per calving/lambing/kidding:</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="IMPEdit" name="Edit5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>OFFPRING</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="Label58">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Calving/lambing interval (days):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="IMPEdit" name="Edit6">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>CALVINTER</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="Label59">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Calving/lambing/kidding rate (% of animals):</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="IMPEdit" name="Edit7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="FieldType">
           <enum>IMPEdit::Numeric</enum>
          </property>
          <property name="TableName">
           <string>lvstosys</string>
          </property>
          <property name="FieldName">
           <string>CALVRATE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMPEdit</class>
   <extends>QLineEdit</extends>
   <header>impedit.h</header>
  </customwidget>
  <customwidget>
   <class>impgenmaint</class>
   <extends>QWidget</extends>
   <header>impgenmaint.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
